rasa/telemetry.py

```diff
@@ -782,6 +782,7 @@ def track_model_training(
             "num_synonyms": len(nlu_data.entity_synonyms),
             "num_regexes": len(nlu_data.regex_features),
             "is_finetuning": is_finetuning,
+            "recipe": config.get("recipe"),
```

Comment:
we do some key accesses to `policies` and `pipeline` a few lines up - what happens if we a use a recipe different than the default?

Reply:
They will be both empty. We can send graph recipe as train and predict instead of pipeline/policy but they are much more verbose so I don't think it's worth it.

Reply:
mhm, might still be interesting though, no? Just in case

Reply:
> mhm, might still be interesting though, no? Just in case

I've added training and predict schemas coming from graph recipe to telemetry now. I don't look at metabase often for this information but could be useful to others.